server/public/styles.css

```diff
@@ -4,20 +4,36 @@ body {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  background-color: #eee;
 }
 
 .container {
   background-color: white;
+  padding: 0;
```

Comment:
Bootstrap has `padding` and `margin` utilities. Try to stick with those to keep your CSS minimal.
https://getbootstrap.com/docs/4.1/utilities/spacing/